### DISCUSSION TITLE
possibility to extract a "one-liner" formula, given an end cell

### DISCUSSION THREAD

**Discussion by dberardo-com** [iddan/react-spreadsheet#311] [Ideas]
It would be useful to get a one liner string representing the concatenation of all formulas, given an "end cell".

so basically having one formula like "layout.resolve_cell('A2')" which basically resolves all references from that cell, till its "leaves dependencies", example:

A2: "=SUM(A1,A3)"
A3: "=SUM(1,3)"
A1: "=A4"
A4: "=SUM(1,1,1,1,1)"

layout.resolve_cell('A2') --> returns: "=SUM(SUM(1,1,1,1,1),SUM(1,3)"
layout.resolve_cell('A1') --> returns: "=SUM(1,1,1,1,1)"

and so on...

is there any existing way to achieve this already ?

**Comment by iddan** [comment]
Hey @dberardo-com, currently there's no way to do such things. I'm open for Pull Requests but currently, I don't have the capacity to implement it myself.
